Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN \
         apt-get purge python3-pip -y &&\
         apt-get autoremove -y &&\
         apt-get clean && \
-        rm -rf /var/lib/apt/lists/ \
+        rm -rf /var/lib/apt/lists
```

Comment:
I'm not sure if it matters but in #61 , it is `https://github.com/NebraLtd/hm-config/pull/61/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R83`